pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.black]
+line-length = 115
+target-version = ['py37']
+include = '\.pyi?$'
+exclude = '''
+
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | \.pytest_cache
+    | _build
+    | buck-out
+    | build
+    | dist
+    | docs
+  )/
+  | versioneer.py
+)
+'''
```

Comment:
Where does this file come from?

Reply:
This is the new "standard" place to put tool related configuration.

Reply:
This is from manual for 'black'. I just changed the default string length and added some directories.